tutorials/21_render_order.md

```diff
@@ -0,0 +1,52 @@
+\page render_order Render order
+
+This example shows how to set the render order for coplanar poligons.
+
+The material API allows changing the render order. When polygons are coplanar, you can get problems
+with `depth fighting` where the pixels from the two polys compete for the same screen pixel. As you
```

Comment:
a few lines down below uses the term `z-fighting` instead of `depth fighting`. I've seen the two terms used interchangeably but we can also make it clear to the users that the two are the same thing, e.g. simple way may just be adding something like `(also known as z fighting)` here